wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -698,11 +705,13 @@ extension WireCallCenterV3 {
 
             Task {
                 do {
+                    // Join the subgroup or create it if it doesn't exist
```

Comment:
Optional: I'm not sure these comments in this task are necessary as it is mainly clear from the methods called, add if we change anything, these comments may fall out of sync.

Reply:
I like them because I feel like it helps separating each step visually, but I agree that they may be redundant with the code since it's pretty self explanatory